client/src/App.js

```diff
@@ -114,8 +114,36 @@ export default class NormalDisplay extends React.Component {
         }
     }
 
+    async getJournalUrl() {
+        if (this.isAborted) return;
+        let resp;
+        try {
+            resp = await fetch("/api/get-journal-url", {
+                method: 'GET',
+                headers: {
+                    Authorization: this.state.devToolToken
+                }
+            });
+            if (!resp.ok) {
+                const errorMessage = await this.formatErrorMessage(resp, 'Error setting up event provider');
+                throw new Error(errorMessage);
+            }
+            resp = await resp.json();
+            console.log(`Asset Compute journal url: ${resp.journalUrl}`);
+            this.setState({
+                journalUrl: resp.journalUrl
+            });
+            document.getElementById('run').disabled = false;
+            return resp.journalUrl;
+        } catch (e) {
+            console.log(e);
```

Comment:
In other places, like for instance, https://github.com/adobe/asset-compute-devtool/blob/f165b4bda8d00ec7be9865dccc1bfe18679995ad/client/src/App.js#L112, a `Log` function is used instead of `console.log`. Please also use it for consistency (if this is just a debug statement, then remove the `console.log`).